cmd/log.go

```diff
@@ -32,4 +32,11 @@ func init() {
 	logCmd.Flags().StringVar(&logOptions.LogPath, "logPath", "stdout", "Output location for alerts and logs, {path|stdout|none}")
 	logCmd.Flags().StringVar(&logOptions.LogFilter, "logFilter", "policy", "Filter for what kinds of alerts and logs to receive, {policy|system|all}")
 	logCmd.Flags().BoolVar(&logOptions.JSON, "json", false, "Flag to print alerts and logs in the JSON format")
+	logCmd.Flags().StringVar(&logOptions.Namespace, "namespace", "", "Specify the namespace")
+	logCmd.Flags().StringVar(&logOptions.Operation, "operation", "", "Give the type of the operation (Eg:Process/File/Network)")
+	logCmd.Flags().StringVar(&logOptions.LogType, "logType", "", "Log type you want (Eg:ContainerLog/HostLog) ")
+	logCmd.Flags().StringVar(&logOptions.ContainerName, "containerName", "", "name of the container ")
+	logCmd.Flags().StringVar(&logOptions.PodName, "podName", "", "name of the pod ")
```

Comment:
```suggestion
	logCmd.Flags().StringVar(&logOptions.ContainerName, "container", "", "name of the container ")
	logCmd.Flags().StringVar(&logOptions.PodName, "pod", "", "name of the pod ")
```
I believe it's sorta obv that we are passing the respective resource name with the flag. WDYT?

Reply:
yeah